mongodb/src/main/java/org/hibernate/ogm/datastore/mongodb/binarystorage/BinaryStorageManager.java

```diff
@@ -0,0 +1,102 @@
+/*
+ * Hibernate OGM, Domain model persistence for NoSQL datastores
+ *
+ * License: GNU Lesser General Public License (LGPL), version 2.1 or later
+ * See the lgpl.txt file in the root directory or <http://www.gnu.org/licenses/lgpl-2.1.html>.
+ */
+package org.hibernate.ogm.datastore.mongodb.binarystorage;
+
+import java.lang.invoke.MethodHandles;
+import java.lang.reflect.Field;
+import java.util.Collections;
+import java.util.EnumMap;
+import java.util.Map;
+
+import org.hibernate.ogm.datastore.mongodb.logging.impl.Log;
+import org.hibernate.ogm.datastore.mongodb.logging.impl.LoggerFactory;
+import org.hibernate.ogm.datastore.mongodb.options.BinaryStorageType;
+import org.hibernate.ogm.datastore.mongodb.options.impl.BinaryStorageOption;
+import org.hibernate.ogm.datastore.mongodb.utils.TableEntityTypeMappingInfo;
+import org.hibernate.ogm.model.key.spi.EntityKey;
+import org.hibernate.ogm.model.spi.Tuple;
+import org.hibernate.ogm.options.spi.OptionsContext;
+import org.hibernate.ogm.options.spi.OptionsService;
+
+import com.mongodb.client.MongoDatabase;
+import org.bson.Document;
+
+/**
+ * @author Sergey Chernolyas &amp;sergey_chernolyas@gmail.com&amp;
+ */
+public class BinaryStorageManager {
+	private static final Log log = LoggerFactory.make( MethodHandles.lookup() );
+
+	private static final Map<BinaryStorageType,BinaryStorageDelegator> BINARY_STORAGE_DELEGATOR_MAP;
+	private static final BinaryStorageDelegator NULL_DELEGATOR = new NullDelegator();
```

Comment:
Instead of `NullDelegator`, let's call it `NoopBynaryStorage`

Reply:
`Noop-` is the prefix we use for things that do No Operations`

Reply:
fixed